zerver/tests/test_signup.py

```diff
@@ -5540,6 +5540,11 @@ def test_dev_user_registration(self) -> None:
         self.assertEqual(result["Location"], "http://zulip.testserver/")
         self.assert_logged_in_user_id(user_profile.id)
 
+    @override_settings(TERMS_OF_SERVICE_VERSION=None, REALM_HOSTS={"zulip": "127.0.0.1"})
+    def test_dev_user_registration_empty_subdomain(self) -> None:
+        result = self.client_post("/devtools/register_user/", subdomain="")
+        self.assertEqual(result["Location"], "http://127.0.0.1/")
```

Comment:
I'm not sure what this is testing, but usually we check `self.assertEqual(result.status_code, 302)` for redirects. 

This should be for a separate PR that only does this one thing, but it wouldn't be a bad idea to have a `self.assert_redirect_to(result, url)` helper that could simplify that pattern -- we apparently have ~300 uses of this pattern according to `git grep status_code | grep 302 | wc`.